### DISCUSSION TITLE
Ensuring Universal Accessibility to GitHub Profile

### DISCUSSION THREAD

**Discussion by sarikarai29** [community/community#111000] [Accessibility]
### Select Topic Area

Question

### Question for GitHub Community Title: Ensuring Universal Accessibility to GitHub Profile 

Body: How can I ensure universal accessibility to my GitHub profile, given that I can access it from my mobile and laptop, but encounter a 404 error from other devices? My GitHub link: [pushpakrai1607](https://github.com/pushpakrai1607) I'm facing difficulty resolving this error. Any guidance or suggestions would be greatly appreciated. Thank you. 

![Screenshot_2024-03-05-18-00-33-13_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/community/community/assets/76905002/a025fe4b-d527-4b22-8b01-00e128dc46da)

**Comment by samus-aran** [comment]
Hi @sarikarai29 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
